Svc/RateGroupDriver/RateGroupDriver.hpp

```diff
@@ -43,17 +68,10 @@ namespace Svc {
             //!
             RateGroupDriver(const char* compName);
 
-            //!  \brief RateGroupDriver initialization function
-            //!
-            //!  The init() function initializes the autocoded base class
-
-            void init(NATIVE_INT_TYPE instanceId = 0);
-
             //!  \brief RateGroupDriver configuration function
```

Comment:
What happened to the init() call? Is the new pattern to call the base class init()?

Reply:
Yes, we've been slowly removing `init` calls for some time.  The base-class method is sufficient. Here is why:

1. `init` is called in autocode by FPP.  Thus custom configuration is discouraged here.  Devs may choose to use the constructor or a dedicated `configure` method that takes in the necessary parameters.
2. `init` is not implemented with `override` and thus often leads to shadowing of the `init` method.